docroot/nbproject/project.xml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@piyuesh23 
It's already added in gitignore.
I have not pushed that file from my local but don't know how it's showing in commits.